src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -120,7 +120,7 @@ export class TechRecordSummaryComponent implements OnInit {
   toggleReasonForCreation(): void {
     if (this.isEditing) {
       this.sectionTemplates.unshift(reasonForCreationSection);
-    } else {
+    } else if (this.sectionTemplates[0] && this.sectionTemplates[0].name === 'reasonForCreationSection') {
```

Comment:
Can this be reduced down to

`else if (this.sectionTemplates[0]?.name === 'reasonForCreationSection')`

Reply:
I can't make JS do what I want ever, is this equivalent? Did this the other day and it still broke lmao